src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -314,9 +314,11 @@ function ChartDeploymentHistory({
                 (selectedDeploymentTabIndex === 2 && !selectedDeploymentManifestDetail.manifest))
         ) {
             return (
-                <CDEmptyState
-                    subtitle={`${deploymentTabs[selectedDeploymentTabIndex]} is not available for this deployment`}
-                />
+                <div className='flex h-100'>
+                    <CDEmptyState
+                        subtitle={`${deploymentTabs[selectedDeploymentTabIndex]} is not available for this deployment`}
```

Comment:
Please move the hardcoded text in the constant messaging > `ERROR_EMPTY_SCREEN`

Reply:
updated 👍